### ISSUE TITLE
"Ooops you don't have permission" screen shown briefly when logging in

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2438]
### Expected behavior
A login screen should not be shown in the main screen while the login is processing

### Actual Behavior
"Ooops you don't have permission" screen is briefly shown while login screen is processing
![login](https://user-images.githubusercontent.com/72819/27163286-eeacaac6-51b8-11e7-9981-e8dc65c8703f.gif)

### Steps to Reproduce the Behavior
1. Open Reaction
1. Login
1. Observe login screen is shown briefly

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.9.3
Reaction: 1.2.0
Reaction branch: ek-ordersPlaceholder
Docker: 17.03.1-ce
```

**Comment by rhenshaw56** [comment]
I believe there is a fix for this on #2459

**Comment by zenweasel** [comment]
@rhenshaw56 That PR was merged. Can you verify this was fixed and close this ticket if it is?
